src/etc/inc/priv.inc

```diff
@@ -269,12 +269,32 @@ function getAllowedPages($username, &$attributes = array()) {
 
 	if ($_SESSION['remoteauth']) {
 		$authcfg = auth_get_authserver($config['system']['webgui']['authmode']);
-		// obtain ldap groups if we are in ldap mode
-		if ($authcfg['type'] == "ldap") {
-			$allowed_groups = @ldap_get_groups($username, $authcfg);
-		} elseif ($authcfg['type'] == "radius") {
-			$allowed_groups = @radius_get_groups($attributes);
-		}
+        	// cache auth results for a short time to prevent hammering auth services & logs
+        	$recheck_time = 30;
```

Comment:
This should be set in an else block on the if conditional below, or use the ternary operator (?:) and do it all in one line.